modules/ROOT/pages/disable-outbound-policies.adoc

```diff
@@ -0,0 +1,77 @@
+= Disable Outbound Policy
```

Comment:
```suggestion
= Disable Outbound Policies
```

Reply:
Due to lack of editing time, copyedited only.
Let's revisit when there is time to work on content and organization.

Reply:
Done! Thanks!